src/recordlinker/schemas/pii.py

```diff
@@ -399,8 +406,9 @@ def feature_iter(self, feature: Feature) -> typing.Iterator[str]:
                 if name.family:
                     yield normalize_text(name.family)
         elif attribute == FeatureAttribute.RACE:
-            if self.race and self.race not in [Race.UNKNOWN, Race.ASKED_UNKNOWN]:
-                yield str(self.race)
+            for race in self.race:
+                if race and race not in [Race.UNKNOWN, Race.ASKED_UNKNOWN]:
+                    yield str(race)
```

Comment:
This change has me second guessing whether we should allow race to be included in `FeatureAttribute`. We don't have log odds for race so we can't use it for blocking or evaluation (assuming we move forward with the log odds window approach and moving blocking to be reliant on log odds as well). 

This is likely beyond the scope of this ticket but it seems that `FeatureAttribute` should be based on the data that is available from the algorithm. If a user has log odds for race, then `FeatureAttribute.RACE` is valid. Thoughts on making `FeatureAttribute` more reliant on the supplied algorithm specification instead of an enum?

Reply:
Oh, that's in the other PR!  Check out this feature https://github.com/CDCgov/RecordLinker/pull/249/files#diff-f8b75e7f2665e1eea4eee28b690b98fa9fd3b2687327a847c80df04145ecb069R229

Reply:
Oh nice, will review that separately!